pygmt/datasets/earth_age.py

```diff
@@ -30,26 +30,28 @@ def load_earth_age(
 
        Earth seafloor crustal age dataset.
 
-    The grids are downloaded to a user data directory
-    (usually ``~/.gmt/server/earth/earth_age/``) the first time you invoke
-    this function. Afterwards, it will load the grid from the data directory.
-    So you'll need an internet connection the first time around.
-
-    These grids can also be accessed by passing in the file name
-    **@earth_age**\_\ *res*\[_\ *reg*] to any grid processing function or
-    plotting method. *res* is the grid resolution (see below), and *reg* is
-    the grid registration type (**p** for pixel registration or **g** for
-    the gridline registration).
-
-    The default color palette table (CPT) for this dataset is *@earth_age.cpt*.
-    It's implicitly used when passing in the file name of the dataset to any
-    grid plotting method if no CPT is explicitly specified. When the dataset
-    is loaded and plotted as an :class:`xarray.DataArray` object, the default
-    CPT is ignored, and GMT's default CPT (*turbo*) is used. To use the
-    dataset-specific CPT, you need to explicitly set ``cmap="@earth_age.cpt"``.
-
-    Refer to :gmt-datasets:`earth-age.html` for more details about available
-    datasets, including version information and references.
+    This function downloads the dataset from the GMT data server, caches it in a user
+    data directory (usually ``~/.gmt/server/earth/earth_age/``), and load the dataset as
+    an :class:`xarray.DataArray`. An internet connection is required the first time
+    around, but subsequent calls will load the dataset from the local data directory.
+
+    The dataset can also be accessed by specifying a file name in any grid processing
+    function or plotting method, using the following file name format:
+    **@earth_age**\_\ *res*\_\ *reg*. *res* is the grid resolution; *reg* is the grid
+    registration type (**p** for pixel registration, **g** for gridline registration).
+    If *reg* is omitted (e.g., ``@earth_age_01d``), the gridline-registered grid will
+    be loaded by default. If *res* is also omitted (i.e., ``@earth_age``), GMT
+    automatically selects a suitable resolution based on the current region and
+    projection settings.
+
+    This dataset comes with a color palette table (CPT) file, ``@earth_age.cpt``. To use
+    the dataset-specific CPT when plotting the dataset, explicitly set
+    ``cmap="@earth_age.cpt"``, otherwise GMT's default CPT (*turbo*) will be used. If
+    the dataset is referenced by file name in a grid plotting method, the
+    dataset-specific CPT file is used automatically unless another CPT is specified.
```

Comment:
```suggestion
    This dataset comes with a color palette table (CPT) file, ``@earth_age.cpt``. To use
    the dataset-specific CPT when plotting the dataset, explicitly set
    ``cmap="@earth_age.cpt"``, otherwise GMT's default CPT (*turbo*) will be used. If
    the dataset is referenced by the file name in a grid plotting method, the
    dataset-specific CPT file is used automatically unless another CPT is specified.
```